crates/store/re_types/definitions/rerun/blueprint/archetypes/force_collision_radius.fbs

```diff
@@ -0,0 +1,15 @@
+namespace rerun.blueprint.archetypes;
+
+/// Resolves collisions between the bounding spheres, according to the radius of the nodes.
+struct ForceCollisionRadius (
+    "attr.rerun.scope": "blueprint"
+) {
+    /// Whether the force is enabled.
+    enabled: rerun.blueprint.components.Enabled ("attr.rerun.component_optional", nullable, order: 100);
+
+    /// The strength of the force.
+    strength: rerun.blueprint.components.ForceStrength ("attr.rerun.component_optional", nullable, order: 200);
+
+    /// The number of iterations to run the force.
```

Comment:
What does that mean exactly? That the force will be active for the N first iterations of the sim for each time step a collision is "active"? (aka dont hesitate to expend the docstring to help build a better mental model of the force behaviour).

```suggestion
    /// The number of iterations to run the force.
    ///
    /// ...
```